src/DynamoPublish/Properties/Resources.resx

```diff
@@ -190,7 +190,7 @@
     <value>Uploading...</value>
   </data>
   <data name="WorkspacesSendSucceededServerResponse" xml:space="preserve">
-    <value>/customize-[0-9,a-z]*</value>
+    <value>/[0-9,a-z]*</value>
```

Comment:
@aosyatnik This is to general. If we change this, let's make it `/share/:id`. This would be more idiomatic of REST Api's.